app/controllers/OERWorldMap.java

```diff
@@ -67,14 +50,24 @@
   Configuration mConf;
   Environment mEnv;
   protected static BaseRepository mBaseRepository = null;
+  protected static Types mTypes;
+  private static ObjectMapper OBJECT_MAPPER = new ObjectMapper();
   static AccountService mAccountService;
   static DatabaseReader mLocationLookup;
   static PageProvider mPageProvider;
 
   private static synchronized void createBaseRepository(Configuration aConf) {
     if (mBaseRepository == null) {
       try {
-        mBaseRepository = new BaseRepository(aConf.underlying(), new ElasticsearchRepository(aConf.underlying()));
+        try {
+          mTypes = new Types(aConf.underlying());
+        } catch (ProcessingException e) {
+          e.printStackTrace();
```

Comment:
I know it's only a minor thing, but we should consistently use the play `Logger`, providing a human readable message along with the exception: `Logger.error("Could not foo bar", e)`

Reply:
ACK! Please see next commit.